### PR TITLE
Minutes for remote users

### LINKED ISSUE #477: Show temporary minutes to remote fsr-users.
I want to see a live version of the latest minute.

### PR DESCRIPTION
**Pull request by Bartzi**
fixes #477 

for information on (hopefully) all steps needed for deploying these changes see [this](https://github.com/fsr-itse/1327/wiki/Deployment) wiki page

### CODE REVIEW

File: _1327/documents/tests.py [hunk 1]
```diff
@@ -895,3 +895,32 @@ def test_delete_button_present_if_editing_already_existing_document(self):
...
+
+	def test_preview_without_get_param(self):
+		response = self.app.get(reverse('documents:preview'), status=404)
+		self.assertEqual(response.status_code, 404)
+
+	def test_preview_view(self):
+		response = self.app.get(reverse('documents:preview') + '?hash_value={}'.format(self.temporary_text.hash_value))
+		self.assertEqual(response.status_code, 200)
+
+		preview_url = '/ws/preview'
```
Comment (janno42) [inline _1327/documents/tests.py]: can we check for content here?

Comment (Bartzi) [inline _1327/documents/tests.py]: what do you mean with check for content?

Comment (janno42) [inline _1327/documents/tests.py]: that the text of the TemporaryDocumentText is shown on the page

Comment (Bartzi) [inline _1327/documents/tests.py]: ehh... yeah that sounds like a reasonable idea^^

**Comment by Bartzi** [comment]
added an assert for the content of the temporary document text in `tests.py`

**Pull request merged**